search/django/Dockerfile

```diff
@@ -1,7 +1,11 @@
 FROM python:3.7
 WORKDIR /code
 COPY ./requirements.txt .
+RUN pip install --upgrade pip
 RUN pip install -r requirements.txt
+ARG weedcoco_version=master
+ADD "http://github.com/Sydney-Informatics-Hub/Weed-AI/archive/"$weedcoco_version".zip" /tmp/weedcoco.zip
```

Comment:
This is quite beautiful.
Docker will request `http://github.com/Sydney-Informatics-Hub/Weed-AI/archive/master.zip` or whatever, and will use the cached layer if it's unchanged!

Reply:
But what does the caching system check now, if it's not the version of the library? From the zip file perspective?

Reply:
Before this PR, it checks if `requirements.txt` has changed.